arbnode/batch_poster.go

```diff
@@ -225,6 +225,7 @@ type BatchPosterConfig struct {
 	AddressMonitorStartL1    uint64 `koanf:"address-monitor-start-l1"`
 	// Please make sure that these addresses are already valid at the `AddressMonitorStartL1`
 	InitBatcherAddresses []common.Address `koanf:"init-batcher-addresses"`
+	IsTimeboosted        bool             `koanf:"is-timeboosted"`
```

Comment:
Should we rename to isDecentralizedTimeboosted? Because there is also centralized timeboost might be confusing